enos/k8s/enos-scenario-k8s.hcl

```diff
@@ -0,0 +1,148 @@
+scenario "k8s" {
+  matrix {
+    edition = ["oss", "ent"]
+  }
+
+  terraform_cli = terraform_cli.default
+  terraform     = terraform.k8s
+
+  providers = [
+    provider.enos.k8s,
+    provider.helm.default,
+  ]
+
+  locals {
+    image_path = abspath(var.vault_docker_image_archive)
+
+    image_repo = var.vault_image_repository != null ? var.vault_image_repository : matrix.edition == "oss" ? "hashicorp/vault" : "hashicorp/vault-enterprise"
+    image_tag = matrix.edition == "oss" ? var.vault_product_version : "${var.vault_product_version}-ent"
+
+    version_includes_build_date = semverconstraint(var.vault_product_version, ">=1.11.0-0")
```

Comment:
Mind throwing in a little comment regarding the `-0` in the constraint?

Reply:
Apparently the range is actually the correct behavior for pre-releases according to the semver spec, so the comment isn't exactly true. What's important is that I don't think most people would realize that without the range using a version that includes pre-release will not do what you think it does.